config/config.go

```diff
@@ -279,10 +279,23 @@ func (c *Config) loadLocalSchema() (*ast.Schema, error) {
 type GenerateConfig struct {
 	Prefix *NamingConfig `yaml:"prefix,omitempty"`
 	Suffix *NamingConfig `yaml:"suffix,omitempty"`
+	Client *bool         `yaml:"client,omitempty"`
```

Comment:
I thought that GenerateClient does not have to be covered with ClientV2

Reply:
This is the config can be read easily like this:

```
generate:
  client: false
```

Which reads quite nice. The default when the `*bool` is `nil` is to generate the client code. It could be a negation if you prefer:

```
generate:
  skip_client: true
``` 

Tell me what you would like to see.

Reply:
it is natural to look at Config. it ok!